astropy/utils/masked/tests/test_function_helpers.py

```diff
@@ -1341,18 +1338,7 @@ def test_nanpercentile(self):
 
 
 untested_functions = set()
-if NUMPY_LT_1_20:
-    financial_functions = {f for f in all_wrapped_functions.values()
-                           if f in np.lib.financial.__dict__.values()}
-    untested_functions |= financial_functions
-
-if NUMPY_LT_1_23:
-    deprecated_functions = {
-        # Deprecated, removed in numpy 1.23
-        np.asscalar, np.alen,
-    }
-else:
-    deprecated_functions = set()
+deprecated_functions = set()
```

Comment:
Is this intentional?

Reply:
Oops, no!! I guess without testing all numpy versions one doesn't see those kinds of things... Now pushed a corrected version.